crates/services/tx_status_manager/src/service.rs

```diff
@@ -223,16 +230,14 @@ impl<Pubkey: ProtocolPublicKey> SignatureVerification<Pubkey> {
 impl<Pubkey: ProtocolPublicKey> Task<Pubkey> {
     fn handle_verified_preconfirmation(
```

Comment:
I suggest renaming this function to `handle_preconfirmations()`.

Previously if was invoked only after `check_preconfirmation_signature()`, but now we can bypass the check by sending `UpdateRequest::Preconfirmations`. Currently, only the executor invokes the `update_preconfirmations()` function, so we're good _for now_.

Anyway, maybe we should have:
`type UntrustedPreconfirmations = Sealed<Preconfirmations, Bytes64>;`
and consider the raw `Preconfirmations` to be always verified.

or something :)

cc: summonig @MitchTurner for opinion :)